packages/trader/src/Modules/CFD/Containers/cfd-reset-password-modal.tsx

```diff
@@ -54,16 +57,22 @@ const CFDResetPasswordModal = ({
     platform,
     setCFDPasswordResetModal,
     history,
-}) => {
-    const [state, setState] = React.useState({
+}: TCFDResetPasswordModal) => {
+    const [state, setState] = React.useState<{
+        error_code: string | number | undefined;
```

Comment:
why do we need string type here?

Reply:
Line 129: response.error.code === 'InvalidToken'. If I remove the string from error_code, TypeScript will show errors on this line. In my opinion it's better to have string as a type for error _code since we might receive some error codes as string too and not just a number;